### PR TITLE
chore(deps): bumping cdk from 1.102.0 to 1.104.0

### PR DESCRIPTION
**Pull request by jusiskin**
## Note to reviewers
Please do not merge yet, as I (@jericht) am still deploying the example app as a smoke test and will merge once that finishes deploying (hopefully) successfully.

## Notes
Bumps CDK version from 1.102.0 to 1.104.0

## Testing

*   Ran `yarn build` from the root which performs a basic RFDK TypeScript build and runs our regression test suite. All tests passed
*   Deployed the `All-In-AWS-Infrastructure-Basic` example app successfully

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by ddneilson** [comment]
Let's do 1.104.0 so that we can get https://github.com/aws/aws-cdk/pull/14431

File: integ/package.json [hunk 1]
```diff
@@ -55,55 +55,55 @@
...
-    "@aws-cdk/aws-logs": "1.102.0",
-    "@aws-cdk/aws-route53": "1.102.0",
-    "@aws-cdk/aws-s3": "1.102.0",
-    "@aws-cdk/aws-s3-assets": "1.102.0",
-    "@aws-cdk/aws-secretsmanager": "1.102.0",
-    "@aws-cdk/core": "1.102.0",
+    "@aws-cdk/aws-docdb": "1.104.0",
+    "@aws-cdk/aws-ec2": "1.104.0",
+    "@aws-cdk/aws-ecr": "1.104.0",
+    "@aws-cdk/aws-autoscaling": "1.104.0",
```
Comment (horsmand) [inline integ/package.json]: The ordering was changed here for some reason so it isn't alphabetical anymore.

File: tools/cdk-build-tools/package.json [hunk 2]
```diff
@@ -35,32 +35,32 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/fs-extra": "^9.0.11",
-    "@types/jest": "^26.0.22",
-    "@types/semver": "^7.3.5",
+    "@types/jest": "^26.0.23",
     "@types/yargs": "^16.0.1",
+    "@types/semver": "^7.3.5",
```
Comment (horsmand) [inline tools/cdk-build-tools/package.json]: Ordering was changed here too so it isn't alphabetical

**Review by horsmand — COMMENTED** [review]
Two little nits that should be quick to fix, otherwise looks good.

File: integ/package.json (hunk 1 above)
Comment (jericht) [reply integ/package.json]: Good catch, not sure why that happened. Might've been changed by `npm-upgrade`.

File: tools/cdk-build-tools/package.json (hunk 2 above)
Comment (jericht) [reply tools/cdk-build-tools/package.json]: Fixed

**Review by ddneilson — DISMISSED** [review]

**Review by horsmand — APPROVED** [review]

**Review by ddneilson — APPROVED** [review]

**Comment by jericht** [comment]
Example app has deployed successfully. Merging...

**Pull request merged**